morebits.js

```diff
@@ -321,7 +325,16 @@ Morebits.quickForm.element.prototype.compute = function QuickFormElementCompute(
 			if (data.label) {
 				label = node.appendChild(document.createElement('label'));
 				label.setAttribute('for', id);
-				label.appendChild(document.createTextNode(data.label));
+				if (!Array.isArray(data.label)) {
+					data.label = [ data.label ];
+				}
+				for (i = 0; i < data.label.length; ++i) {
+					if (typeof data.label[i] === 'string') {
+						label.appendChild(document.createTextNode(data.label[i]));
+					} else if (data.label[i] instanceof Element) {
```

Comment:
```suggestion
					} else if (data.label[i] instanceof Node) {
```

Something like this PR turns out to be necessary for i18n (since some modules like unlink are using HTML in labels using the array form - but for i18n we need a single string that can be put into the messages json file).

Changing `Element` to `Node` will take care of that by allowing use of $.parseHTML in the modules.

That is,
```js
label: $.parseHTML(msg('msg-name'))
```
here `msg-name` can be an html string. Say `hello <strong>world</strong>` which when passed through $.parseHTML gives `[ textNode, strongElement ]`. textNode is instance of `Node` but not `Element`.

I think we can avoid jquery for now, and pull this duplicated code into a function.